MVVMIOS/ViewModel/UserViewModel.swift

```diff
@@ -24,15 +24,24 @@ final class UserViewModel {
 
     // MARK: Fetching data from network.
 
+    var fetchCompleted: (([UserListViewModel]?, _ errorReason: String?) -> Void)?
```

Comment:
Storing a completion like this can be dangerous as it can lead to memory cycles. Probably easier to pass the completion into `fetchFromNetwork` and call it directly inside there so you know it isn't being retained anywhere.

Reply:
Sorry, I was experimenting with things on that side, so it was left there. I shall take of this now on.